.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "vendor/googletest"]
```

Comment:
Not sure if we want keep dependencies as submodules, but for now it can stay as it is.